packages/node-sdk/src/client.ts

```diff
@@ -657,11 +655,10 @@ export class BucketClient {
     await this._config.batchBuffer.flush();
   }
 
-  private _getFeatures({
-    enableTracking = true,
-    ...context
-  }: ContextWithTracking): Record<string, RawFeature> {
-    void this.syncContext({ enableTracking, ...context });
+  private _getFeatures(
```

Comment:
Not related to your PR but I really dislike the fact that we have two notations for private members: one uses underscore and the other doesn't. We should use only one. Also I typically document private ones as well so we either do in both case or remove docs from all privates.

Reply:
agreed, will cleanup separately.